src/pages/project-pitches/components/tabs/PitchOverviewTab.tsx

```diff
@@ -32,14 +35,44 @@ const PitchOverviewTab = ({ pitch }: PitchOverviewTabProps) => {
 
   const { openModal } = useModalContext();
   const { format } = useDate();
+  const mapFunctions = useMap();
   const onEdit = () => openModal("pitchEditModal", <PitchEditModal pitch={pitch} />);
+  const [polygonBbox, setPolygonBbox] = useState<any>(null);
+  const [polygonDataMap, setPolygonDataMap] = useState<any>({});
+
+  const { data: projectPolygon } = useGetV2TerrafundProjectPolygon(
+    {
+      queryParams: {
+        entityType: "project-pitch",
+        uuid: pitch.uuid ?? ""
+      }
+    },
+    {
+      enabled: !!pitch.uuid
+    }
+  );
 
-  let projectBoundary: any;
-  try {
-    projectBoundary = JSON.parse(pitch.proj_boundary!);
-  } catch (e) {
-    projectBoundary = undefined;
-  }
+  const setBbboxAndZoom = async () => {
+    if (projectPolygon?.project_polygon?.poly_uuid) {
+      const bbox = await fetchGetV2TerrafundPolygonBboxUuid({
+        pathParams: { uuid: projectPolygon.project_polygon?.poly_uuid ?? "" }
+      });
+      const bounds: any = bbox.bbox;
+      setPolygonBbox(bounds);
+    }
+  };
+
+  useEffect(() => {
+    const getDataProjectPolygon = async () => {
+      if (!projectPolygon?.project_polygon) {
+        setPolygonDataMap({ [FORM_POLYGONS]: [] });
+      } else {
+        setBbboxAndZoom();
+        setPolygonDataMap({ [FORM_POLYGONS]: [projectPolygon?.project_polygon?.poly_uuid] });
+      }
```

Comment:
nitpick: can you handle the true case first?

Reply:
great, done